src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -317,7 +317,12 @@
                         </td>
                         <td>
                             {% if decision.attributes|length == 1 %}
-                                {{ decision.attributes|first }}
+                                {% set first = decision.attributes|first %}
+                                {% if first.expression is defined %}
+                                    Expression: <code><pre>{{ first.expression }}</pre></code>
```

Comment:
should be `<pre><code></code></pre>` as `<code>` is an inline element while `<pre>` is a block element.